spec/lib/api/2.0/ibms-spec.js

```diff
@@ -65,26 +65,15 @@ describe('Http.Api.Ibms', function () {
         }
     };
 
-    before('start HTTP server', function () {
-        this.timeout(5000);
-        return helper.startServer().then(function() {
-            waterline = helper.injector.get('Services.Waterline');
-            Errors = helper.injector.get('Errors');helper.injector.get("Services.Configuration");
-            configuration = helper.injector.get("Services.Configuration");
-            //sinon.stub(configuration);
-        });
-    });
+    helper.httpServerBefore();
 
-    afterEach(function () {
-        if (stub) {
-            stub.restore();
-            stub = undefined;
-        }
+    before(function () {
+        waterline = helper.injector.get('Services.Waterline');
+        Errors = helper.injector.get('Errors');helper.injector.get("Services.Configuration");
```

Comment:
Good finding, it's not my purpose here. It's from the original code, and I think we should delete it.